netmiko/base_connection.py

```diff
@@ -1634,6 +1649,8 @@ def send_command(
         if normalize:
             command_string = self.normalize_cmd(command_string)
 
+        log.info("In send_command, cmd:{}, read_timeout:{}".format(command_string, read_timeout))
```

Comment:
related to below code, we should change _prompt_handler to raise exception if find_prompt raises exception. this is what we changed in netmiko2 as well

```
        if expect_string is not None:
            search_pattern = expect_string
        else:
            search_pattern = self._prompt_handler(auto_find_prompt)

Reply:
in netmiko2 we added a log.info statement when the exception was swallowed 
https://github.com/cafykit/netmiko/blob/7fcc795484e4ccdecc7b3e8b6e365b1cc5c7afb7/netmiko/base_connection.py#L1095

Reply:
added the similar log.info line in netmiko4

Reply:
created seperate PR for find_prompt() changes to amke it independent https://github.com/cafykit/netmiko-v4/pull/15